spec/plainyearmonth.html

```diff
@@ -269,9 +269,14 @@ <h1>Temporal.PlainYearMonth.prototype.add ( _temporalDurationLike_ [ , _options_
           1. Let _day_ be 1.
         1. Let _date_ be ? CreateTemporalDate(_yearMonth_.[[ISOYear]], _yearMonth_.[[ISOMonth]], _day_, _calendar_).
         1. Let _durationToAdd_ be ? CreateTemporalDuration(_duration_.[[Years]], _duration_.[[Months]], _duration_.[[Weeks]], _balanceResult_.[[Days]], 0, 0, 0, 0, 0, 0).
-        1. Let _addedDate_ be ? CalendarDateAdd(_calendar_, _date_, _durationToAdd_, _options_).
+        1. Let _optionsCopy_ be ! OrdinaryObjectCreate(*null*).
+        1. Let _overflow_ be ? ToTemporalOverflow(_options_).
+        1. Perform ! CreateDataPropertyOrThrow(_optionsCopy_, *"overflow"*, _overflow_).
+        1. Let _addedDate_ be ? CalendarDateAdd(_calendar_, _date_, _durationToAdd_, _optionsCopy_).
```

Comment:
For PYM.add and subtract, I think it'd be better to copy the whole options object (example in [MergeLargestUnitOption](https://tc39.es/proposal-temporal/#sec-temporal-mergelargestunitoption)) and pass the original to CalendarDateAdd and the copy to YearMonthFromFields.